src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -88,8 +88,24 @@ int LuaCreateFloatingWindow(lua_State *L) {
     try {
         if (withImgui) {
             wnd = std::make_shared<ImGUIWindow>(width, height, decoration);
+            XPLMWindowID window = wnd->getXWindow();
+
+            std::ostringstream oss_window_id;
+            oss_window_id << "FlyWithLua Info: This XPLMWindowID " << window << " is for LuaCreateFloatingWindow";
+            std::string window_id;
+            window_id = oss_window_id.str();
+
+            flywithlua::logMsg(logToAll, window_id);
```

Comment:
This can be simplified to:
```cpp
std::string window_id = oss_window_id.str();
flywithlua::logMsg(logToAll, window_id);
```
...or even to:
```cpp
auto window_id = oss_window_id.str();
flywithlua::logMsg(logToAll, window_id);
```
...or even to:
```cpp
flywithlua::logMsg(logToAll, oss_window_id.str());
```

However, a true C++ way (which I'm mostly saying for completeness would be exposing logging as an output stream of its own, similar to std::cout. Unfortunately, it'd be a bit cumbersome to implement it on top of C interface of X-Plane SDK, so I'm not suggesting doing that now.

Same below.